.github/workflows/devcontainer.yml

```
@@ -0,0 +1,137 @@
+            echo "::error::Each component must match [a-z0-9]+([._-][a-z0-9]+)* separated by /"
+            exit 1
+          fi
+          
+          IMAGE_NAME="${{ steps.registry.outputs.registry }}/${{ steps.repo_owner.outputs.owner_lc }}/$IMAGE_NAME"
+          echo "✅ Final image name: $IMAGE_NAME"
+          echo "image_name=$IMAGE_NAME" >> $GITHUB_OUTPUT 
+      
+      - name: Sanitize Image Tag
+        if: steps.check.outputs.exists == 'true'
+        id: sanitized_tag
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:2:40: Double quote to prevent globbing and word splitting

(shellcheck)